core-backend/common/src/funcs.rs

```diff
@@ -255,6 +255,36 @@ pub fn send_push<E: Ext>(ext: LaterExt<E>) -> impl Fn(i32, i32, i32) -> Result<(
     }
 }
 
+pub fn create_program<E: Ext>(
+    ext: LaterExt<E>,
+) -> impl Fn(i32, i32, i32, i32, i32, i64, i32, i32) -> Result<(), &'static str> {
+    move |code_hash_ptr: i32,
+          salt_ptr: i32,
+          salt_len: i32,
+          payload_ptr: i32,
+          payload_len: i32,
+          gas_limit: i64,
```

Comment:
We are to split this method into two: with `gas_limit` and without. Maybe later.

Reply:
Shall I leave the current one?

Reply:
I think that's not the problem. We will gone out of gas limits only after #715, so it may be done later

Reply:
I guess, resolve this

Reply:
We can rename it to `create_program_wgas`.

Reply:
Maybe better later create `create_program_gas_less`?
Or default is create without gas?

Reply:
See `send_wgas` etc in #715

Reply:
Done